ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -64,7 +64,7 @@
 import io.confluent.ksql.rest.integration.RestIntegrationTestUtil;
 import io.confluent.ksql.rest.server.TestKsqlRestApp;
 import io.confluent.ksql.schema.ksql.LogicalSchema;
-import io.confluent.ksql.schema.ksql.PhysicalSchema;
+import io.confluent.ksql.serde.PhysicalSchema;
```

Comment:
Can you move this back to the `io.confluent.ksql.schema.ksql` package please?

Reply:
You mean leave PhysicalSchema in ksqldb-common but keep the old package? That makes sense, will do.